src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs

```diff
@@ -82,6 +87,7 @@ public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, s
                 var mode = arguments.Named("mode");
                 var quality = arguments.Named("quality");
                 var format = arguments.Named("format");
+                var appendVersion = arguments.Named("appendVersion");
```

Comment:
I'd call this in Shortcode "append_version", to follow the lead of `asset_url` (the only case where an existing shortcode has a two-word term) and [the Liquid equivalent](https://docs.orchardcore.net/en/latest/docs/reference/modules/Media/#append_version).